backend/diary/models.py

```
@@ -188,14 +175,34 @@ def parsed_planting_period(self):
             end_day, end_month = end_str.split(".")
 
             today = date.today()
-            start_date = date(today.year, romanian_months[start_month], int(start_day))
-            end_date = date(today.year, romanian_months[end_month], int(end_day))
+            start_date = date(today.year, roman_months[start_month], int(start_day))
+            end_date = date(today.year, roman_months[end_month], int(end_day))
 
             return start_date, end_date
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)